Source/WebKit/Platform/spi/mac/AppKitSPI.h

```diff
@@ -97,3 +97,14 @@ typedef void (^NSWindowSnapshotReadinessHandler) (void);
 - (NSWindowSnapshotReadinessHandler)_holdResizeSnapshotWithReason:(NSString *)reason;
 @end
 #endif
+
+#if USE(APPLE_INTERNAL_SDK)
+#import <AppKit/NSMenu_Private.h>
+#else
+@interface NSKeyboardShortcut
+
++ (id)shortcutWithKeyEquivalent:(NSString *)keyEquivalent modifierMask:(NSUInteger)modifierMask;
+@property (readonly) NSString *localizedDisplayName;
+
+@end
+#endif
```

Comment:
Should group this with the imports above and the existing `#if USE(APPLE_INTERNAL_SDK)`, and the corresponding `#else`.

Reply:
Missed that!